apps/script-service/Services/ScriptFunctions.cs

```diff
@@ -73,17 +73,17 @@ public ScriptFunctions(AdspId tenantId, IServiceDirectory directory, Func<Task<s
     return result?.Configuration;
   }
 
-  public FormDataResult? GetFormData(string formId)
+  public virtual FormSubmissionResult? GetFormSubmission(string formId, string submissionId)
```

Comment:
Are you getting rid of an existing functionality here as well?

Also what does virtual do in this case?

Reply:
No, that was a copy-paste problem.  Another good catch.

Reply:
Actually, it is needed for the override in StubScriptFunctions.